solitude/settings/sites/paymentsalt/db.py

```diff
@@ -17,6 +17,7 @@
 DATABASES['default'] = dj_database_url.parse(private.DATABASES_DEFAULT_URL)
 DATABASES['default']['ENGINE'] = 'django.db.backends.mysql'
 DATABASES['default']['OPTIONS'] = {'init_command': 'SET storage_engine=InnoDB'}
+DATABASES['default']['ATOMIC_REQUESTS'] = True
```

Comment:
hmm, this isn't very DRY. I suppose it will do for now. Seems like `DATABASES['default'] = base.DATABASES['default'].copy()` should be possible though.